### PR TITLE
new image icon interface

### PR DESCRIPTION
**Pull request by stevecrozz**
Refs #784

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://stevecrozz-new-icon-interface.23f39035947abfbf29f9f7741cb9c271.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://stevecrozz-new-icon-interface.23f39035947abfbf29f9f7741cb9c271.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: app/assets/stylesheets/issue_tracker_icons.css.erb [hunk 1]
```diff
@@ -1,9 +0,0 @@
-/* Issue Tracker inactive, select, create and goto icons */
-
-<% ErrbitPlugin::Registry.issue_trackers.keys.each do |label| %>
-div.issue_tracker.nested label.<%= label %> { background: url(<%= asset_path "#{ label }_inactive.png" %>) no-repeat; }
-div.issue_tracker.nested label.r_on.<%= label %> { background: url(<%= asset_path "#{ label }_create.png" %>) no-repeat; }
-#action-bar a.<%= label %>_create { background: transparent url(<%= asset_path "#{ label }_create.png" %>) 6px 5px no-repeat; }
-#action-bar a.<%= label %>_goto { background: transparent url(<%= asset_path "#{ label }_goto.png" %>) 6px 5px no-repeat; }
-<% end %>
-
```
Comment (arthurnn) [inline app/assets/stylesheets/issue_tracker_icons.css.erb]: :fire:

File: Gemfile.lock [hunk 2]
```diff
@@ -213,7 +213,7 @@ GEM
       jwt (~> 0.1.4)
       multi_json (~> 1.0)
       rack (~> 1.2)
-    octokit (3.7.0)
+    octokit (3.8.0)
```
Comment (arthurnn) [inline Gemfile.lock]: why updating octokit here too?

**Comment by arthurnn** [comment]
code looks great.. Dont have time to test that, but I would say :ship:

File: app/assets/stylesheets/issue_tracker_icons.css.erb (hunk 1 above)
Comment (stevecrozz) [inline app/assets/stylesheets/issue_tracker_icons.css.erb]: lol. my sentiments exactly

File: Gemfile.lock (hunk 2 above)
Comment (stevecrozz) [inline Gemfile.lock]: that's a mistake. good catch

**Comment by stevecrozz** [comment]
This was much trickier than I originally thought. But it should be about ready now.

**Pull request merged**